apps/src/templates/TooltipOverlay.jsx

```diff
@@ -160,9 +160,22 @@ export function textProvider(label) {
  * Simple provider that formats and renders the mouse coordinates.
  * @returns {function(): string}
  */
-export function coordinatesProvider(flip) {
+export function coordinatesProvider(flip = false, isRtl = false) {
   return props => {
     const y = flip ? props.height - props.mouseY : props.mouseY;
-    return `x: ${Math.round(props.mouseX)}, y: ${Math.round(y)}`;
+    if (isRtl) {
+      return [
+        '\u202A',
+        Math.round(y),
+        ' :y',
+        ', ',
+        Math.round(props.mouseX),
+        ' :',
+        'x',
+        '\u202C'
+      ].join(''); // We have to use unicode to explicitly set the SVG text to LTR when the documen is using RTL
```

Comment:
I'd much prefer to see this implemented using interpolation

```suggestion
      // We have to use unicode to explicitly set the SVG text to LTR when the document is using RTL
      return `\u202A${Math.round(y)} :y, ${Math.round(props.mouseX)} :x\u202C`;
```

Reply:
@Hamms when I was testing the unicode it doesn't change when using interpolation with backticks. I could do concatenation with + but not sure how much of an improvement that is

Reply:
hmmm, that shouldn't be the case. Quick manual testing indicates that those two approaches produce completely identical strings:

```js
one = ['\u202A', 1, ' :y', ', ', 2, ' :', 'x', '\u202C'].join('');
two = `\u202A${1} :y, ${2} :x\u202C`;

console.log(one === two);
```

What behavior difference do you see when using interpolation vs concatenation?